### PR TITLE
Add namespace context for exec command in the test

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
The test applies oc commands without the name of the namespace. 

### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-5065
https://github.com/eclipse/che/issues/22575

### How to test this PR?:
Set the env. Where will be a test run for the Namespace/OpenShift project.: For example, 
`export TEST_NAMESPACE=admin-devspaces`
Review and merge PR, get results from the Jenkins pipeline

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by SkorikSergey — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
@musienko-maxim : should PR fix the issue https://github.com/eclipse/che/issues/22575 ?

Asking, because `WorkspaceWithParent` test had failed on "_Check expected containers in the parent POD_" step again being run using this branch: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/16241/consoleFull

**Comment by musienko-maxim** [comment]
Yes, we can close the issue because this PR should fix the https://github.com/eclipse/che/issues/22575. If we get some other problems,  I hope it won't happen - we can create new issue

**Comment by dmytro-ndp** [comment]
@musienko-maxim : I don't think we can close the issue https://github.com/eclipse/che/issues/22575, because the test has failed: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/16241/consoleFull

File: tests/e2e/specs/miscellaneous/WorkspaceWithParent.spec.ts [hunk 1]
```diff
@@ -79,23 +77,23 @@ suite(`Workspace using a parent test suite ${BASE_TEST_CONSTANTS.TEST_ENVIRONMEN
...
-			`${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} get pod ${podName} --output jsonpath=\'{.spec.initContainers[].name}\'`
+			`${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} get pod ${podName} -n ${BASE_TEST_CONSTANTS.TEST_NAMESPACE} --output jsonpath=\'{.spec.initContainers[].name}\'`
 		);
 		expect(initContainerName).contains('che-code-injector');
 	});
 
 	test('Check expected environment variables', function (): void {
 		const envList: string = shellExecutor.executeArbitraryShellScript(
-			`${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} exec -i ${podName} -c tools -- sh -c env`
+			`${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} -n admin-devspaces exec -i ${podName} -c tools -- sh -c env`
```
Comment (dmytro-ndp) [inline tests/e2e/specs/miscellaneous/WorkspaceWithParent.spec.ts:96]: `admin-devspaces` is hardcoded

**Pull request merged**